website/docusaurus.config.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
   organizationName: 'facebook',
   projectName: 'docusaurus',
   baseUrl,
+  baseUrlIssueBanner: true,
```

Comment:
Why specify this if it is the default value?

Reply:
I just find it convenient sometimes to have the full list of options already in the config to play with, so that I don't have to look up in the doc ;)